pkg/utils/utils.go

```diff
@@ -258,3 +286,18 @@ func CleanCachedNetConf(cRefPath string) error {
 	}
 	return nil
 }
+
+// IsValidMACAddress checks if net.HardwareAddr is a valid MAC address.
+func IsValidMACAddress(addr net.HardwareAddr) bool {
+	return len(addr) == 6
```

Comment:
should we also add a regex validation here?

Reply:
net.HardwareAddr tyoe is just a list of bytes.
maybe just check its not all zeroes ?

Reply:
I added a check for zeros and broadcast. I hope this addresses your concern @SchSeba